lib/noticed/base.rb

```diff
@@ -67,23 +67,31 @@ def params
 
     private
 
-    # Runs all delivery methods for a notification
-    def run_delivery(recipient, enqueue: true)
-      delivery_methods = self.class.delivery_methods.dup
+    def run_all_deliveries(recipients, enqueue: true)
+      individual_delivery_methods = self.class.delivery_methods.dup
```

Comment:
This method is where we will eventually be segregating between individual delivery methods and bulk delivery methods based on the `options` provided.

After the segregation, we will eventually have one more method below `run_individual_deliveries`, named `run_bulk_deliveries `